development/Adding-a-new-platform-source.md

```diff
@@ -1,17 +1,17 @@
-Thank you for help for the development of *Chat Overflow*.  
-While plugins live in their own project, source connections (e.g. to platform like Twitch or Discord) exist in the framework itself.  
-By this, we can make sure that everyone profits from the new possibilites.  
+Thank you for helping developing *Chat Overflow*.  
+While plugins live in their own project, source connections (e.g. to a platform like Twitch or Discord) exist in the framework itself.  
+By this measure, we can make sure that everyone profits from the same features.  
```

Comment:
by this measure still sounds weird, what about "Because of this"

Reply:
Line 3 also does not convey the same point as before.

Reply:
The whole sentence in line 3 is kind of weird. If I understand the original sentence correctly, the platform sources (the things that enable you to connect to the platforms) are baked into the framework itself. And if I remember correctly, this is done in order to prevent that someone develops a proprietary plugin/extension? So by bringing all the platform connectors to the framework, everybody has the same access to them.

I can agree that both the original sentence as well as the new sentence is weird, but I can't make it much better if I don't know for certain what is meant by the original.

Reply:
Proprietary plugins are explicitly allowed: 
>It's getting even better: The Chat Overflow license allows you to sell your custom plugins, creating new services for other streamers.  

_[README.md](https://github.com/codeoverflow-org/chatoverflow/blob/master/README.md)_

Let me explain it in german in order to avoid misunderstandings:
Mit Plugins kann jeder sich mit den sersichedenen Dienesten verbinden und unter verwendung der API einfach eigene Chatbots, Abläufe oder ähnliches programmieren.

Die Schnittstellen zu den Diensten (wie Twitch, Discord, ...) müssen fest im Framework verankert sein, sodass jeder Plugin Entwickler einfach darauf zugreifen kann. 
Das erlaubt es das jeder alle Features mit eigenen Plugins nutzen kann.
Gleichzeitig wären Proprietäre Plugins theoretisch möglich.

Reply:
Also Schnittstellen zu Diensten sind im Framework. Verstanden.
Das ist so, damit jeder Entwickler zugriff hat.
Folglich gibt es keine proprietary Dienst-Schnittstellen, richtig?

Hab ich oben auch mit plugin/extension gemeint, im nachhinein ist das aber echt eine schlechte Wortwahl von mir gewesen. Sorry!

Aber hab ich's jz verstanden?

Reply:
Exakt 👍